src/ui/components/results/resultsheadercomponent.js

```diff
@@ -9,7 +9,11 @@ const DEFAULT_CONFIG = {
   showFieldNames: false,
   resultsCountSeparator: '|',
   verticalURL: undefined,
-  showChangeFilters: false
+  showChangeFilters: false,
+  hiddenFields: ['builtin.location'],
+  removable: false, // TODO implement
+  delimiter: '|',
```

Comment:
removed any mention of DEPRECATED and now only use new config names for vertical results. for results header config names were not changed because this is a component designed purely for internal use